### ISSUE TITLE
'Ignore' button when renaming clusters

### ISSUE THREAD

**Issue by Derkades** [matiasdelellis/facerecognition#375]
A button to permanently ignore a face group (never ask to give it a name in the future)

**Comment by matiasdelellis** [comment]
Yes.. It's a bit annoying .. but at this stage I wanted to minimize the changes in database..
Soon i will work on this..

Thanks.. 😄

**Comment by DerOetzi** [comment]
I would prefer such option. In addition to this there should be a possibility to ignore such faces at clustering as well to speed up this process. But there should be the possibility to reshow ignored faces.

**Comment by trvrcr** [comment]
I would prefer to remove single faces from a cluster, 
The clusters are quite OK in grouping the correct persons.
But sometimes there is 1 wrong picture/face.
So then the cluster is useless.
